### PR TITLE
Add one line description for missing Scaladoc tags

### PR DESCRIPTION
**Pull request by janekdb**
WIP pushed for early review comments.

The initial objective is to have at least a one line description for each tag.

### CODE REVIEW

**Comment by janekdb** [comment]
@VladUreche - Babystep as mentioned. Pushed for early feedback. Thanks!

**Comment by VladUreche** [comment]
LGTM, thanks @janekdb! Should I leave the PR open so you can continue adding things, or do you prefer each baby step to be merged individually?

**Comment by janekdb** [comment]
@VladUreche - please leave open until this first step is done. My preference for this activity will be to have many small PRs so we can get this in front of users quicker.

File: overviews/scaladoc/for-library-authors.md [hunk 1]
```diff
@@ -89,6 +89,24 @@ include:
   definition is too complex or noisy. An example is (in the collections API),
   providing documentation for methods that omit the implicit `canBuildFrom`.
 
+
+### Member grouping tags
+- `@group <group>` - marks the entity as a member of the `<group>` group.
+- `@groupname <group> <name>` - provide an optional name for the group. `<name>` is displayed as the group header
+-  before the group description.
+- `@groupdesc <group> <description>` - supply descriptive text to include under the group name. Supports multiline
+   formatted text. `<description>` is displayed under the group name.
```
Comment (janekdb) [inline overviews/scaladoc/for-library-authors.md]: Repetition.

File: overviews/scaladoc/for-library-authors.md [hunk 2]
```diff
@@ -89,6 +89,26 @@ include:
...
+
+### Member grouping tags
+- `@group <group>` - mark the entity as a member of the `<group>` group.
+- `@groupname <group> <name>` - provide an optional name for the group. `<name>` is displayed as the group header
+-  before the group description.
+- `@groupdesc <group> <description>` - add optional descriptive text to display under the group name. Supports multiline
+   formatted text.
+- `@groupprio` - control the order of the group on the page. Defaults to 0. Ungrouped elements have
+  an implicit priority of 1000. Use a value between 0 and 999 to set a relative position to other groups. Low values
+  will appear before high values.
```
Comment (janekdb) [inline overviews/scaladoc/for-library-authors.md]: This is known as _Inversion of Priority_ :)

File: overviews/scaladoc/for-library-authors.md [hunk 3]
```diff
@@ -89,6 +89,26 @@ include:
...
+
+### Member grouping tags
+- `@group <group>` - mark the entity as a member of the `<group>` group.
+- `@groupname <group> <name>` - provide an optional name for the group. `<name>` is displayed as the group header
+-  before the group description.
+- `@groupdesc <group> <description>` - add optional descriptive text to display under the group name. Supports multiline
+   formatted text.
+- `@groupprio` <priority> - control the order of the group on the page. Defaults to 0. Ungrouped elements have
+  an implicit priority of 1000. Use a value between 0 and 999 to set a relative position to other groups. Low values
+  will appear before high values.
```
Comment (janekdb) [inline overviews/scaladoc/for-library-authors.md]: This is known as _Inversion of Priority_ :)

**Comment by SethTisue** [comment]
@janekdb should we go ahead and merge?

**Comment by janekdb** [comment]
@SethTisue - Yes, please merge. On balance there is more value to merging than not.

I can follow up the outstanding items later. They are marked with `TODO` and `???`.

**Pull request merged**